### PR TITLE
Fix status code for GET /replica, when replica is starting

### PR DESCRIPTION
**Pull request by anikin-aa**
https://github.com/zalando/patroni/issues/1128

### CODE REVIEW

File: patroni/api.py [hunk 1]
```diff
@@ -109,7 +109,7 @@ def do_GET(self, write_status_code_only=False):
         elif 'master' in path or 'leader' in path or 'primary' in path or 'read-write' in path:
             status_code = primary_status_code
         elif 'replica' in path:
-            status_code = replica_status_code
+            status_code = 200 if response.get('state') == 'running' else 503
```
Comment (CyberDem0n) [inline patroni/api.py]: Your change fixes the problem only partially and unfortunately not 100% correctly.
Instead you should look on the line [104](https://github.com/zalando/patroni/pull/1152/files#diff-790530624eaa21e5fb614fc321aef277R104) and fix it.

Comment (anikin-aa) [reply patroni/api.py]: ok, got it, my bad. changes are commited

**Comment by sdudoladov** [comment]
:+1:

**Comment by CyberDem0n** [comment]
:+1:

**Pull request merged**